html/settings/_settings.scss

```diff
@@ -199,7 +199,7 @@ $sprk-horizontal-list-spacing-medium: $sprk-space-m !default;
 $sprk-horizontal-list-spacing-large: $sprk-space-l !default;
 /// The huge spacing value for huge Horizontal List components.
 $sprk-horizontal-list-spacing-huge: $sprk-space-xl !default;
-/// The delimiter between delimited items in Horizonatal List components.
```

Comment:
Haha good typo catch. I had to look twice to see what the change was.

Reply:
I actually can't take credit for catching these typos. I have a spell check extension for my code editor. 🤫